dev/ledger.md

```diff
@@ -1030,13 +1040,15 @@ point must be discarded and the entire transaction rejected.
 
 ### Application Stateful TEAL Execution Semantics
 
-- During the execution of an `ApprovalProgram` or `ClearStateProgram`, the
-  application’s `LocalStateSchema` and `GlobalStateSchema` may never be
-  violated. The program's execution will fail on the first instruction that
-  would cause the relevant schema to be violated. Writing a `Bytes` value to a
-  local or global [TEAL Key/Value Store][TEAL Key/Value Stores] longer than 64
-  bytes, or writing any value to a key longer than 64 bytes, will likewise cause
-  the program to fail on the offending instruction.
+- During the execution of an `ApprovalProgram` or `ClearStateProgram`,
+  the application’s `LocalStateSchema` and `GlobalStateSchema` may
+  never be violated. The program's execution will fail on the first
+  instruction that would cause the relevant schema to be
+  violated. Writing a `Bytes` value to a local or global [TEAL
+  Key/Value Store][TEAL Key/Value Stores] such that the sum of the
+  lengths of the key and value in bytes exceeds 128, or writing any
```

Comment:
Is 128 a consensus parameter?

Reply:
Yes.  Should I name it here?  We don't seem to do that in ledger.md, where there are several such (maximum schema sizes, program costs, app argument count and size).

Reply:
I think it would be best to always name them. But no need to do it in this commit. This can be a later commit cleaning names.